teuthology/suite/__init__.py

```diff
@@ -200,7 +206,47 @@ def get_rerun_conf_overrides(conf):
         )
         return
 
-    conf.filter_in.extend(rerun_filters['descriptions'])
+    if getattr(conf, 'rerun_skip_known', False):
+        unknown_descriptions = filter_unknown_failures(run, conf.rerun_statuses)
```

Comment:
It makes misunderstandings, what are the unknowns descriptions? All the descriptions are known.